### DISCUSSION TITLE
PDF export not containing online only meetings

### DISCUSSION THREAD

**Discussion by christpage** [code4recovery/12-step-meeting-list#1372] [Q&A] [answered]
I am using the Export utility to generate a pdf for a local meeting schedule book.  The pdf being generated does not contain any online only meetings.  Is there a way to include these meetings?

**Comment by gkovats** [comment, marked as answer]
Currently the utility is only for generating a listing of in-person meetings only. The tool doesn't online-only, inactive, or by-appointment meetings.

**Comment by dougie-wougie** [comment]
I asked the same question, and the group response made sense. Why would there be an online meeting in a printed document? Users would have to type in the Zoom codes. On our printed book, I added a QR code linked to the meetings in the Meeting Guide plugin on our Central Office site. 
![CleanShot 2024-02-26 at 06 41 20](https://github.com/code4recovery/12-step-meeting-list/assets/58369176/320bd73e-1b9c-4cfc-ad56-a2fa2287f9f4)

**Reply by joshreisner** [reply]
🙏 this is the way
